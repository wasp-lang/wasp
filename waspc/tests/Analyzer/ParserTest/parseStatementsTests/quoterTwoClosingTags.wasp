test TwoClosingTags {=foo foo=} foo=}
