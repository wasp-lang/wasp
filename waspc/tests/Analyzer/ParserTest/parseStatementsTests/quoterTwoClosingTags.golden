Parse error at 1:36:
  Unexpected token: =
  Expected one of the following tokens instead: <identifier>

Parse error at 1:37:
  Unexpected token: }
  Expected one of the following tokens instead: <identifier>,<eof>
